it/auth.json

```diff
@@ -23,262 +76,260 @@
+      "no_organization_access": "Spiacenti, non hai accesso a questa organizzazione.",
+      "operation_not_permitted": "L'azione richiesta non può essere completata.",
+      "invitation_email_mismatch": "Quell'indirizzo email non sembra corretto. Dovrai accedere usando l'indirizzo email a cui è stato inviato l'invito.",
+      "email_domain_not_allowed": "Non possiamo completare la tua registrazione poiché il dominio email rilevato non è consentito. Prova un altro metodo di registrazione.",
+      "invitation_expired": "Sembra che tu abbia aspettato un po' troppo per accettare e il tuo invito è scaduto. Chiedi al tuo amministratore di invitarti di nuovo.",
+      "user_suspended": "Non puoi accedere al momento",
+      "unknown_error": "Qualcosa è andato storto quando abbiamo cercato di autenticarti e non possiamo offrire una soluzione rapida. Inizia una nuova sessione e prova ad accedere di nuovo."
+    },
+    "account_not_found_page": {
+      "page_title": "Account non trovato",
+      "heading": "Non riusciamo a trovare il tuo account",
+      "description": "Abbiamo bisogno di alcuni dettagli in più per finalizzare il tuo account.",
+      "description_sign_ups_disabled": "Prova a usare un account diverso o un'altra opzione di accesso",
+      "description_social": "Vuoi crearne uno nuovo con il tuo account ${social_provider_name}?",
+      "description_credential": "Vuoi creare un nuovo account con ${credential} ora?",
+      "continue_button": "Crea account",
+      "back_button": "Torna all'accesso",
+      "sign_in_another_way_separator": "oppure",
+      "sign_in_another_way_link": "accedi in un altro modo"
+    },
+    "phone_number_component": {
+      "phone_input_label": "Numero di telefono",
+      "country_select_label": "Paese",
+      "use_email_link": "Passa all'email",
+      "use_phone_link": "Passa al numero di telefono",
+      "phone_input_error_sms_limit_reached": "Impossibile inviare il codice. Servizio di messaggi SMS non configurato. Accedi usando un altro metodo.",
+      "phone_input_error_invalid_country_code": "Prefisso internazionale non valido. Seleziona dalla lista dei paesi.",
+      "phone_input_error_required": "Il numero di telefono non può essere lasciato vuoto",
+      "phone_input_error_invalid_format": "Inserisci un numero valido senza spazi o prefisso internazionale",
+      "phone_input_error_no_account_found": "Nessun account trovato con questo numero di telefono"
+    },
+    "username_component": {
+      "username_input_label": "Nome utente",
+      "username_input_error_required": "Il nome utente non può essere lasciato vuoto",
+      "username_input_error_invalid": "Inserisci un nome utente valido",
+      "username_input_error_no_account_found": "Nessun account trovato con questo nome utente",
+      "username_input_error_invalid_format": "Spiacenti. I nomi utente possono contenere solo lettere, numeri, trattini (-) o underscore (_).",
```

Comment:
Spiacenti. I nomi utente possono contenere solo lettere, numeri, trattini (-) o trattini bassi (_).

Reply:
"underscore" is not translated to Italian here. It should be "trattini bassi"

Reply:
In Italy underscore is widely used also.